examples/proxy/envelope-encryption/base/kafka/kafka-persistent.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+  namespace: kafka
+spec:
+  kafka:
+    version: 3.6.0
```

Comment:
Is that right? Do we want `3.6.0` or `3.6.1`?

Reply:
It'll be updated to 3.7 before it goes out.  I notice that the examples still reference `3.6.0`.

https://github.com/k-wall/amqstreams-1-openshift-image/blob/ENTMQSTPR-14/examples/kafka/kafka-ephemeral-single.yaml